### ISSUE TITLE
Question about  pretrained kinetic400 model accuracy?

### ISSUE THREAD

**Issue by sean186** [MCG-NJU/TDN#44]
Hi, thanks for your awesome work in video recognition and also the release.

Recently, I tested the pretrained [kinetic400 checkpoint](https://drive.google.com/drive/folders/1UALrg2ABFq-HGPdy8vgZ9mxqkSkuqWET) on 19799 videos by executing the provided test command, and only got 53% top-1 precision, which is a little bit far from the reported 76.6%.  It bothered me a lot and the dataset in my drive seems fine. 
Here are some my test dataset file 
```
sword_fighting/_kcVbo4E2JQ_000101_000111 300 345 
bowling/eU7Ht6zJcyk_000003_000013 142 31   
celebrating/bUidM8i-buc_000012_000022 300 51    
pumping_fist/OjowI-z7-oo_000006_000016 270 256
```

Some test logs:
```video 896 done, total 896/19799, average 0.544 sec/video, moving Prec@1 52.478 Prec@5 74.138
video 928 done, total 928/19799, average 0.534 sec/video, moving Prec@1 52.396 Prec@5 73.958
video 960 done, total 960/19799, average 0.577 sec/video, moving Prec@1 52.823 Prec@5 74.294
video 992 done, total 992/19799, average 0.562 sec/video, moving Prec@1 52.539 Prec@5 74.414
video 1024 done, total 1024/19799, average 0.547 sec/video, moving Prec@1 52.557 Prec@5 74.527
video 1056 done, total 1056/19799, average 0.532 sec/video, moving Prec@1 52.941 Prec@5 74.816
video 1088 done, total 1088/19799, average 0.567 sec/video, moving Prec@1 52.857 Prec@5 74.643
video 1120 done, total 1120/19799, average 0.554 sec/video, moving Prec@1 53.212 Prec@5 74.740
video 1152 done, total 1152/19799, average 0.541 sec/video, moving Prec@1 53.209 Prec@5 74.662
video 1184 done, total 1184/19799, average 0.530 sec/video, moving Prec@1 52.878 Prec@5 74.753
video 1216 done, total 1216/19799, average 0.550 sec/video, moving Prec@1 53.125 Prec@5 75.080
```

So could you please help me to figure it out? thx
(ps, I used the original data, and it was not scaled to 320px)

**Comment by yztongzhan** [comment]
hi @sean186, thanks for your question.
Please give me for details such as the command to help you debug :)
Besides, others have tested our model in #2 and the results are only a bit lower than ours, because their source video data is 256p. So, you may check the code and pre-trained model to make sure the testing scheme is right.

**Comment by sean186** [comment]
> hi @sean186, thanks for your question. Please give me more details to help you debug, such as your command :) Besides, others have tested our model in #2 and the results are only a little bit lower than ours, because their source video data is 256p and different from ours. So, you may check the code and pre-trained model to make sure your testing scheme is right.

Thanks for your reply. It was work when I re-downloaded the [kinetic400_resnet50_segment16 ](https://drive.google.com/drive/folders/1p7NC4JjxaLzwfj1JUa4kkHd_KZBxBBYR)model，and got Prec@1 77.17% Prec@5 93.08% with shorter side 256p imgs. 
Maybe there is a problem with the 8 segments model.

**Comment by yztongzhan** [comment]
Thanks for your update and we would check the pre-trained model later.
